src/components/Map/Map.tsx

```diff
@@ -158,7 +159,7 @@ export default function Map() {
     }
   );
 
-  const { query } = useRouter();
+  const router = useRouter();
```

Comment:
Why remove the destructuration ? (It is not a problem, just my curiosity)

Reply:
because i need to acces the 'push' method, and extracting a push method without context seems weird